thamos/config.py

```diff
@@ -217,7 +217,7 @@ def api_discovery(self, host: str = None) -> str:
                 )
         except Exception:
             # Try without TLS - maybe router was not configured to use TLS.
-            api_url = "http://" + host + "/api/v1"
+            api_url = urljoin("https://" + host, "/api/v1")
```

Comment:
This part expects a route without TLS being turned on (hence "http://" not "http**s**://) - see the comment one line above.

@harshad16 could you please fix this?

Reply:
...merged... I didnt catch that, sorry.

Reply:
@goern @fridex my apologize :(
will raise the pr for the fix. @fridex